src/app/main/app-routing.module.ts

```diff
@@ -2,22 +2,18 @@ import { NgModule } from "@angular/core";
 import { RouterModule } from "@angular/router";
 import { GetComponent } from "./get.component";
 import { ConnectComponent } from "connect/connect.component";
-import { HomeComponent } from "./home.component";
 import { NotFound } from "common/notfound.component";
 import { WebServerRoute } from "./settings";
 
 @NgModule({
   imports: [
     RouterModule.forRoot([
-        // site only routes
-        { path: "", component: HomeComponent },
-
         // common routes
+        { path: "", redirectTo: "/webserver", pathMatch: "full" },

```

Comment:
Make sure web server/services still works after this change.

Reply:
This PR is to unblock fixing some accessibility issue. We will fix this page if we decide to proceed with
https://github.com/microsoft/IIS.WebManager/issues/219